Documentation/install/upgrade.rst

```diff
@@ -316,6 +316,24 @@ Annotations:
 1.7 Upgrade Notes
 -----------------
 
+1.7.13 Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new flag ``--egress-multi-home-ip-rule-compat`` was introduced to allow
```

Comment:
I wonder if it'd be better to lead with the fix, then describe the flag? EG:

> In rare cases, Cilium 1.7.12 or earlier when configured with ENI IPAM mode
could disrupt connectivity for the node when new ENI devices are created on
the node. This version fixes the issue and introduces a new flag to handle compatibility.